index.js

```diff
@@ -0,0 +1,161 @@
+const from = require('from2')
+const { EventEmitter } = require('events')
+
+const { Api, JsonRpc } = require('eosjs')
+const { JsSignatureProvider } = require('eosjs/dist/eosjs-jssig') // development only
+const fetch = require('node-fetch') // node only; not needed in browsers
+const { TextEncoder, TextDecoder } = require('util') // node only; native TextEncoder/Decoder
+
+module.exports = configure
+
+function configure (opts) {
+  if (!opts) opts = {}
+  const chainId = opts.chainId || '5fff1dae8dc8e2fc4d5b23b2c7665c97f9e9d8edf2b6485a86ba311c25639191'
+  const rpc = new JsonRpc(opts.rpc || 'https://api-kylin.eoslaomao.com', { fetch })
+
+  let api = null
+  const account = opts.account
+  if (!account) throw new Error('opts.account must be set')
+
+  if (opts.privateKey) {
+    const signatureProvider = new JsSignatureProvider([ opts.privateKey ])
+
+    api = new Api({
+      rpc,
+      signatureProvider,
+      textDecoder: new TextDecoder(),
+      textEncoder: new TextEncoder(),
+      chainId
+    })
+  }
+
+  return {
+    pay,
+    subscription,
+    createTransactionStream
+  }
+
+  function pay (destination, amount, memo, cb) {
+    if (!api) throw new Error('opts.privateKey must be provided in the constructor')
+
+    api.transact({
+      actions: [{
+        account: 'eosio.token',
+        name: 'transfer',
+        authorization: [{
+          actor: account,
+          permission: 'owner'
```

Comment:
owner permission is like root - usually active is used for transfers

Reply:
i'll change it. should be configurable also i'm guessing?

Reply:
yes, that would be cool.